lib/seek/util.rb

```diff
@@ -180,5 +180,11 @@ def self.filter_disabled(types)
       types.select(&:feature_enabled?)
     end
 
+    def self.extended_metadata_supported_types
+      ActiveRecord::Base.descendants.select do |model|
```

Comment:
you need to be sure the models have been loaded by the classloader first. You can do this by using `persistent_classes`. Also cache. Have a look at the other methods, like authorized_types - the pattern is the same just calling a different method